Library/Homebrew/github_packages.rb

```diff
@@ -45,13 +45,65 @@ def upload_bottles(bottles_hash)
       skopeo = Formula["skopeo"].opt_bin/"skopeo"
     end
 
+    # TODO: these dependencies are installed but cannot be required automatically.
+    Homebrew.install_gem!("public_suffix")
+    Homebrew.install_gem!("addressable")
+    Homebrew.install_gem!("json-schema")
+    require "json-schema"
```

Comment:
This gem is no longer maintained btw: https://github.com/ruby-json-schema/json-schema/issues/423#issuecomment-507206304

Reply:
@Bo98 Argh. I'm going to merge this for now given the other improvements but I'll move to the new gem (likely next week).